src/app/hearing-part/components/hearing-search-table/hearing-search-table.component.ts

```diff
@@ -68,4 +69,9 @@ export class HearingSearchTableComponent implements OnInit, OnChanges {
     buildViewHearingUrl(id: string) {
         return `/home/hearing/${id}`;
     }
+
+    canEdit(hearing: FilteredHearingViewmodel) {
+        return hearing.status === Status.Unlisted || (hearing.status === Status.Listed
+            && moment(moment(hearing.listingDate).format('YYYY-MM-DD')).isAfter(moment(moment().format('YYYY-MM-DD'))));
```

Comment:
1. why casting it to string and from string back to date
2. moment(moment().format('YYYY-MM-DD')) if you have to do it like this, you can do just 
```suggestion
            && moment(moment(hearing.listingDate).format('YYYY-MM-DD')).isAfter(moment().format('YYYY-MM-DD')));
```
I think I know what you would like to do. (I'm not sure if listingDate is a string or moment. I would assume that it is a string
```suggestion
            && moment(hearing.listingDate).startOf('day').isAfter(moment().startOf('day'));
```